package.json

```diff
@@ -7,6 +7,7 @@
     "scripts": {
         "testrpc": "testrpc -p 8545 --networkId 50 -m \"${npm_package_config_mnemonic}\"",
         "lerna:run": "lerna run",
+        "lerna:rebuild": "lerna run clean; lerna run build;",
```

Comment:
`lerna run clean && lerna run build`
As far as I'm aware `lerna run clean` asks for a user confirmation.
Maybe `yes | lerna run clean`?

Reply:
No it doesn't. You are thinking of `lerna clean` ;)